packages/js/components/src/product-fields/fields/checkbox/render.tsx

```diff
@@ -20,7 +20,7 @@ const CheckboxField: React.FC< CheckboxFieldProps > = ( {
 		<CheckboxControl
 			label={ label }
 			onChange={ onChange }
-			selected={ value }
+			checked={ value }
```

Comment:
`selected` didn't even exist on `CheckboxControl`. Reason this was never noticed is because we don't use this code. We can actually remove the entire `product-fields` directory, this was also part of the first iteration of the product editor ( pre block editor era ). Will do this as a follow up.